src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5109,6 +5109,6 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
     <value>Unable to read debug information of method '{0}' (token 0x{1:X8}) from assembly '{2}'</value>
   </data>
   <data name="ERR_DynamicLocalFunctionTypeParameter" xml:space="preserve">
-    <value>Cannot pass argument with dynamic type to generic parameter '{0}' of local function '{1}'.</value>
+    <value>Cannot pass argument with dynamic type to generic local function '{0}'. Try specifying the type arguments explicitly.</value>
```

Comment:
How about "Cannot pass argument with dynamic type to generic local function with inferred type arguments"?